.dockerignore

```diff
@@ -0,0 +1 @@
+.venv
```

Comment:
do you want for example result.png be present in docker image? let's ignore all redundant files